### ISSUE TITLE
Small markup issues

### ISSUE THREAD

**Issue by marcoscaceres** [sysapps/runtime#53]
There are some small markup issues in #12 that should be fixed independently of that bug. 

```
1374c1374
<             as the first argument is a Function object rather than a string;<il>

---
>             as the first argument is a Function object rather than a string;</li>
1377c1377
<             myElement.onXXX = someFunction;<li>

---
>             myElement.onXXX = someFunction;</li>
```

Should also run the current spec over [validator.nu](http://validator.nu).

**Comment by marcoscaceres** [comment]
Work started...

**Comment by marcoscaceres** [comment]
Fixed by merge #54 .
